actionpack/test/dispatch/host_authorization_test.rb

```diff
@@ -83,7 +92,7 @@ class HostAuthorizationTest < ActionDispatch::IntegrationTest
     }
 
     assert_response :forbidden
-    assert_match "Blocked host: www.example.local", response.body
+    assert_empty response.body
```

Comment:
@p8 
I changed all those body assertions because the fact a request is local is not checked anymore.
I think that it's not a problem. 

But if you think differently, I'm ready to change it back and pass `"action_dispatch.show_detailed_exceptions" => true` within the env

Reply:
Ah yes, I think we want to keep these assertions for regression.